src/main/scala/com/crobox/clickhouse/partitioning/PartitionDateFormatter.scala

```diff
@@ -0,0 +1,16 @@
+package com.crobox.clickhouse.partitioning
+
+import java.util.Date
+
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
+
+object PartitionDateFormatter {
```

Comment:
don't get why it's `Partition`. Because for (monthly) partitions format is YYYYMM